clients/include/flops.hpp

```diff
@@ -120,6 +120,48 @@ constexpr double geam_gflop_count(rocblas_int m, rocblas_int n)
     return (3.0 * m * n) / 1e9;
 }
 
+/* \brief floating point counts of TRSM */
+template <typename T>
+constexpr double trmm_gflop_count(rocblas_int m, rocblas_int n, rocblas_side side)
+{
+    if(rocblas_side_left == side)
+    {
+        return (1.0 * m * n * (m + 1)) / 1e9;
+    }
+    else
+    {
+        return (1.0 * m * n * (n + 1)) / 1e9;
+    }
+}
+
+template <>
+constexpr double
+    trmm_gflop_count<rocblas_float_complex>(rocblas_int m, rocblas_int n, rocblas_side side)
+{
+    if(rocblas_side_left == side)
+    {
+        return (1.0 * m * n * (m + 1)) / 1e9;
```

Comment:
I'm thinking the complex flops should be 4x the real ones if I'm not mistaken. Doesn't really matter since they're not implemented yet.

Come to think of it, I likely missed complex calcs for other functions (I'm thinking dot (4x), scal(2x), axpy(2x)?).

Reply:
I don't think I updated flops to be accurate either.   complex mult is 4 fp mult + 2 fp add.   I thought these were only O(?) approximations but if we want to plot them we probably should update them all to be more accurate in a consistent way.

Reply:
Thanks, daineAMD is correct, trmm complex flops is 4x real flops. For most of BLAS flops is from (multiply_add), and complex (multiply_add) has 4 additions and 4 multiplications, so 4x the flops compared to read (multiply_add). I need to check, but I think it is dot(4x), cscal(6x), daxpy(4x).

Reply:
I thought FLOPS is not an instruction count but operation count.  Malcolm has suggested not reporting FLOPS and only report time to avoid these issues.  Can discuss at meeting.

Reply:
Since at least 28-August-1989 with file sblas2time.f from http://www.netlib.org/blas/old-index.html BLAS has used flops/sec to measure BLAS performance. GEMM usually gets close to peak performance. It may easier to understand a flops/sec plot with the peak performance plotted as a line rather than a time result.